pkg/bosh/converter/container_factory.go

```diff
@@ -377,13 +377,7 @@ func boshPreStartInitContainer(
 	}
 }
 
-func bpmPreStartInitContainer(
-	process bpm.Process,
-	jobImage string,
-	volumeMounts []corev1.VolumeMount,
-	debug bool,
-) corev1.Container {
-
+func bpmPreStartInitContainer(process bpm.Process, jobImage string, volumeMounts []corev1.VolumeMount, debug bool) corev1.Container {
```

Comment:
Please, avoid those long lines. It's much easier to read a multi-line parameter list. And also much easier to review parameters added/removed.